patches/tModLoader/Terraria/ModLoader/EquipLoader.cs

```diff
@@ -241,6 +241,131 @@ internal static int GetPlayerEquip(Player player, EquipType type)
 				_ => 0,
 			};
 
+		/// <summary>
+		/// Adds an equipment texture of the specified type, internal name, and associated item to your mod.<br/>
+		/// You can then get the ID for your texture by calling EquipLoader.GetEquipTexture, and using the EquipTexture's Slot property.<br/>
+		/// Returns the ID / slot that is assigned to the equipment texture.
+		/// </summary>
+		/// <param name="mod">The mod the equipment texture is from.</param>
+		/// <param name="item">The item.</param>
+		/// <param name="type">The type.</param>
+		/// <param name="texture">The texture.</param>
+		/// <returns></returns>
+		public static int AddEquipTexture(Mod mod, ModItem item, EquipType type, string texture) {
```

Comment:
The following methods could shorten their duplicate XML lines by using
```cs
/// <inheritdoc cref="AddEquipTexture(Mod, EquipTexture, ModItem, string, EquipType, string)"/>
```

and then only declaring parameters that are missing there.

❗  Also, to be discussed: Whether the Mod parameter is needed, while it could be taken from ModItem.